HoloPipelines/pipelines/components/compHttpRequest.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import pathlib
+import sys
+
+
+def sendFilePostRequest(url, inputFile, outputFile):
+    inputFile = str(pathlib.Path(inputFile))
+    outputFile = str(pathlib.Path(outputFile))
+    with open(inputFile, "rb") as inputFileData:
+        file = {"file": inputFileData}
```

Comment:
change variable name "file" it's somewhat of a restricted keyword

Reply:
renamed to `fileToSend`